src/containerapp/azext_containerapp/_sdk_models.py

```diff
@@ -3910,6 +3912,7 @@ class ContainerResources(_serialization.Model):
         "cpu": {"key": "cpu", "type": "float"},
         "memory": {"key": "memory", "type": "str"},
         "ephemeral_storage": {"key": "ephemeralStorage", "type": "str"},
+        "gpu": {"key": "gpu", "type": "float"},
```

Comment:
Do we supported gpu in all regions?
For `_sdk_models`, it is for `--yaml`, please add test for create/update `gpu` with `--yaml`. 
You can refer test case `test_containerapp_create_with_yaml`